### PR TITLE
Added Form Validation Pair Programming with Than Myo Htet

### PR DESCRIPTION
**Pull request by HtetWaiYan7191**
### Adding Form Validation

- add a required attribute for each input field
- add name attribute for email and name 
- specify max length for name and textarea fields

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -437,7 +437,7 @@
     background: #fff;
     border: 1px solid #cfd8dc;
     border-radius: 8px;
-    padding: 15px 38px 15px 0;
```
Comment (zieeco) [inline style.css:440]: ### CONTACT SECTION

- Hey, the padding left and right is 24px. please, kindly update this part of your style. `#contact-me-section`


*your design*
Screenshot from 2023-05-02 10-34-08

*Figma design*
image

File: index.html [hunk 2]
```diff
@@ -258,10 +258,10 @@ <h2>Contact me</h2>
...
 
-            <form action="" id="contact-me-form">
-                <input type="text" name="user-name">
-                <input type="text" name="email">
-                <textarea name="message-box" id="message-box" cols="30" rows="8" placeholder="Write you message here..."></textarea>
+            <form action="https://formspree.io/f/mknayjll" id="contact-me-form" method="POST">
+                <input type="text" name="user-name" maxlength="30" placeholder="Full Name" required>
+                <input type="email" name="email" placeholder="Email address" required>
+                <textarea name="message-box" maxlength="500" id="message-box" cols="30" rows="8" placeholder="Write you message here..." required></textarea>
                 <button type="submit">Get in touch</button>
```
Comment (zieeco) [inline index.html:265]: - Hey, let's add the missing interaction states of each button in your webpage to indicate the (hover, disabled, enabled, active) to improve the user experience when the buttons are clicked:

image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,







Great job so far! 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -258,10 +258,10 @@ <h2>Contact me</h2>
                     I’d love to help with it!</p>
             </header>
 
-            <form action="" id="contact-me-form">
-                <input type="text" name="user-name">
-                <input type="text" name="email">
```
Comment (ghost) [inline index.html:263]: - [ ] As requested earlier, there is still a bigger space  ( left and right) on your form section. Kindly adjust the padding to match the original design

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @HtetWaiYan7191, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**